Helper/FeatureSwitch/Manager.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Helper\FeatureSwitch;
+
+
+use Magento\Framework\App\Helper\Context;
+use Bolt\Boltpay\Helper\GraphQL\Client as GQL;
+use Bolt\Boltpay\Model\FeatureSwitchRepository;
+use Magento\Framework\App\Helper\AbstractHelper;
+
+class Manager extends AbstractHelper {
+
+    /**
+     * @var GQL
+     */
+    private $gql;
+
+    /**
+     * @var fsRepo
+     */
+    private $fsRepo;
+
+    /**
+     * @param Context $context
+     * @param GQL $gql
+     * @param FeatureSwitchRepository $fsRepo
+     * @codeCoverageIgnore
+     */
+    public function __construct(
+        Context $context,
+        GQL $gql,
+        FeatureSwitchRepository $fsRepo
+    ) {
+        parent::__construct($context);
+        $this->gql = $gql;
+        $this->fsRepo = $fsRepo;
+    }
+
+    /**
+     * This method gets feature switches from Bolt and updates the local DB with
+     * the latest values. To be used in upgrade data and webhooks.
+     *
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function updateSwitchesFromBolt() {
+        $switchesResponse = $this->gql->getFeatureSwitches();
+
+        $switches = array();
+        if ($switchesResponse->getData()) {
+            $data = $switchesResponse->getData();
+            if (isset($data["response"])
```

Comment:
IMO this is redundunt, I think it's ok to assume some contract (response has specific format in case of successful response)

Reply:
i sort of agree, but also think would make our code bullet proof just incase someone by mistake changes bolt then we have to revert. this just makes us bullet proof and saves us from random errors. now given that we have the stfu operator id rather it be this way. please unresolve if you care a lot about this